apps/examples/st_things/Kconfig

```diff
@@ -0,0 +1,29 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_ST_THINGS
+	bool "SmartThings Things example"
+	default n
+	depends on ST_THINGS
+	depends on ENABLE_IOTIVITY
+	depends on ARCH_BOARD_ARTIK053
```

Comment:
Do you want to run this only on artik053? As I left comments before, st_things are a common module, it should be working on any board. Isn't it?

Reply:
Many functions are removed, but that changes are not reflected to the header files. I will remove unused header files. Sorry. 
Also I will change Kconfig after modifying the sample.